### PR TITLE
Paddlecreation2

### LINKED ISSUE #22965: Paddle.tensor creation
PaddleTensor creation

-[]#15024

### PR DESCRIPTION
**Pull request by Novendobi**
# PR random uniform

## Related Issue  paddle.tensor creation 

Close #-[] 22965

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/paddle/tensor/creation.py [hunk 1]
```diff
@@ -144,6 +144,15 @@ def ones_like(x, /, *, dtype=None, name=None):
     return ivy.ones_like(x, dtype=dtype)
 
 
+# []-22965
+@with_supported_dtypes("paddle")
```
Comment (umairjavaid) [inline ivy/functional/frontends/paddle/tensor/creation.py:148]: Your with_supported_dtypes implementation is not correct, this is causing the tests to fail. You have to check the paddle framework version and the crossponding dtypes supported for this function.

**Review by umairjavaid — CHANGES_REQUESTED** [review]

**Comment by Novendobi** [comment]
> Make sure the tests are passing, once you have fixed the supported dtype decorator. I'll be happy to re-review it =)

I have put the correct dtype syncing with its version

**Review by umairjavaid — CHANGES_REQUESTED** [review]
The tests are not passing. You need to pass stratergies for the low and high positional arguments 

```
E   TypeError: random_uniform() missing 2 required positional arguments: 'low' and 'high'
E   Falsifying example: test_paddle_random_uniform(
E       on_device='cpu',
E       backend_fw='jax',
E       frontend='paddle',
E       dtype_and_shape=(['int64'], [array(0)], ()),
E       test_flags=FrontendFunctionTestFlags(
E           num_positional_args=0,
E           with_out=False,
E           inplace=False,
E           as_variable=[False],
E           native_arrays=[False],
E           test_compile=False,
E           generate_frontend_arrays=False,
E           transpile=False,
E           precision_mode=False,
E       ),
E       fn_tree='ivy.functional.frontends.paddle.random_uniform',
E   )
E   
E   You can reproduce this example by temporarily adding @reproduce_failure('6.84.1', b'AXicY2DAAAAAEwAB') as a decorator on your test case
```

**Pull request closed without merging**